Extension/src/LanguageServer/client.ts

```diff
@@ -402,6 +403,10 @@ class DefaultClient implements Client {
             if (changedSettings["commentContinuationPatterns"]) {
                 updateLanguageConfigurations();
             }
+            if (changedSettings["clang_format_path"]) {
+                let settings: CppSettings = new CppSettings(this.RootUri);
+                this.languageClient.sendNotification(UpdateClangFormatPathNotification, util.resolveVariables(settings.clangFormatPath, null));
```

Comment:
Can you just use `changedSettings["clang_format_path"]` without creating a new `CppSettings` object?

Reply:
No, the value is "...".

Reply:
😆 ok